r_markdown/load_save_data.Rmd

```diff
@@ -178,79 +179,91 @@ Reference for test id, stove and fuel types.
+                                      "NA")))
 ```
 
 * Define fuel category levels
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
   samples <- dplyr::mutate(samples,
-                           fuel = factor(fuel,
-                           levels = c("Douglas Fir",
-                             "Oak",
-                             "Eucalyptus",
-                             "Lump Hardwood (Sm.)",
-                             "Lump Hardwood (Med.)",
-                             "Coconut Charcoal",
-                             "West Slope Pellets",
-                             "Eucalyptus Pellets",
-                             "Kerosene",
-                             "Liquefied Petroleum Gas")))
+                           fuel = forcats::lvls_revalue(fuel,
+                                  c("Douglas Fir",
+                                  "Oak",
+                                  "Eucalyptus",
+                                  "Lump Hardwood (Sm.)",
+                                  "Lump Hardwood (Med.)",
+                                  "Coconut Charcoal",
+                                  "West Slope Pellets",
+                                  "Eucalyptus Pellets",
+                                  "Kerosene",
+                                  "Liquefied Petroleum Gas",
+                                  "NA")))
 ```
 
-* Create aggregated fuel category
+* Define stove fuel combination category levels
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
   samples <- dplyr::mutate(samples,
-                           fuelcat =
-                             fct_collapse(fuel,
-                               wood = c("Oak",
-                                        "Eucalyptus",
-                                        "Douglas Fir"),
-                               charcoal = c("Coconut Charcoal",
-                                            "Lump Hardwood (Med.)", 
-                                            "Lump Hardwood (Sm.)"),
-                               pellets = c("Eucalyptus Pellets",
-                                           "West Slope Pellets"),
-                               advanced = c("Kerosene",
-                                            "Liquefied Petroleum Gas")))
+                           stove_fuel = forcats::as_factor(paste(stove, ":", fuel))) %>%
+                           dplyr::mutate(stove_fuel = 
+                           forcats::lvls_reorder(stove_fuel,
+                                                 c(28, 30, 29, 7, 9, 8,
+                                                   1, 3, 2, 25, 27, 26,
+                                                   10, 12, 11, 6, 5, 4,
+                                                   21, 20, 19, 18, 17, 14,
+                                                   13, 16, 15, 31, 23, 24, 22))) # set logical factor order
```

Comment:
i feel like you should be able to do this once and then when you make the additional categories (like this one and the broader ones) you won't need to reorder?